pkg/local-storage/member/node/manager.go

```diff
@@ -364,6 +365,34 @@ func (m *manager) handleVolumeReplicaUpdate(oldObj, newObj interface{}) {
 	m.storageMgr.UpdateNodeForVolumeReplica(replica)
 }
 
+func (m *manager) handleVolumeReplicaDelete(obj interface{}) {
+	replica, _ := obj.(*apisv1alpha1.LocalVolumeReplica)
+	if replica.Spec.NodeName != m.name {
+		return
+	}
+
+	m.logger.WithFields(log.Fields{"replica": replica.Name}).Info("Observed a VolumeReplica CRD deletion...")
+	if replica.Status.State != apisv1alpha1.VolumeReplicaStateDeleted {
+		// must be deleted by a mistake, rebuild it
+		m.logger.WithFields(log.Fields{"replica": replica.Name, "spec": replica.Spec, "status": replica.Status}).Warning("Rebuilding VolumeReplica CRD ...")
+		// TODO: need retry considering the case of creating failure??
+		newReplica := &apisv1alpha1.LocalVolumeReplica{}
+		newReplica.Name = replica.Name
+		newReplica.Spec = replica.Spec
+		newReplica.Status = replica.Status
+
+		if err := m.apiClient.Create(context.TODO(), newReplica); err != nil {
+			m.logger.WithFields(log.Fields{"replica": replica.Name}).WithError(err).Error("Failed to rebuild VolumeReplica")
+		}
+		if err := m.apiClient.Status().Update(context.TODO(), newReplica); err != nil {
+			m.logger.WithFields(log.Fields{"replica": replica.Name}).WithError(err).Error("Failed to rebuild VolumeReplica's statis")
+		}
+	} else {
+		delete(m.replicaRecords, replica.Spec.VolumeName)
+		m.storageMgr.UpdateNodeForVolumeReplica(replica)
```

Comment:
It seems that the Update event of VolumeReplica Status(Deleted)  has been captured in here: https://github.com/hwameistor/hwameistor/blob/83e0c576ce91980f18dbd8b49bb43309d6f3b54d/pkg/local-storage/member/node/manager.go#L211